lib/reek/spec/should_reek_of.rb

```diff
@@ -8,6 +8,7 @@ module Spec
     # An rspec matcher that matches when the +actual+ has the specified
     # code smell.
     #
+    # :reek:InstanceVariableAssumption
```

Comment:
Well, it is nearly always set, not just under odd circumstances. I suppose we could set it directly rather than using the memoization in line 52. Anyway, I don't like the code in this class much, but its design is a bit forced by RSpec's API.